main.scss

```diff
@@ -1,14 +1,183 @@
+/*
+ * NOTE:
+ * This file is used to apply our custom theme to Coral
+ * Talk V5 service.
+ *
+ * The stylesheet is built by the Origami Build Service
+ * and the URL for the CSS file is used in Coral Talk
+ * admin panel as the custom CSS URL.
+ *
+ * This is not a normal use case of Origami styles.
+ */
+
+// sass-lint:disable no-qualifying-elements nesting-depth no-important class-name-format
 @import 'src/scss/variables';
 @import 'src/scss/functions';
 @import 'src/scss/mixins';
 
+@import 'o-buttons/main';
+@import 'o-fonts/main';
+@import 'o-colors/main';
+@import 'o-typography/main';
+
 @if ($o-comments-is-silent == false) {
-	@include oComments();
+	@include oComments;
 
 	// Set to silent again to avoid being output twice
 	$o-comments-is-silent: true !global;
 
-	.talk-plugin-infobox-info {
-		background: pink;
+	@include oFontsInclude(MetricWeb, $style: italic);
```

Comment:
Our components don't usually include font faces individually as they'll be duplicated and the font could change in o-typography. Instead add `o-fonts` to your build service request.

Reply:
Thanks for the feedback. We are currently testing this manually (not using Origami build service). I will keep them here for now and will remove before this PR is ready to merge.